docs/sources/mimir/configure/configure-native-histograms-ingestion.md

```diff
@@ -40,6 +40,6 @@ overrides:
     max_native_histogram_buckets: 160
 ```
 
-To learn more about sending native histograms to Mimir or Grafana Cloud Metrics via Grafana Agent or Prometheus,
-see [Scrape and send native histograms with Grafana Agent]({{< relref "../send/native-histograms#scrape-and-send-native-histograms-with-grafana-agent" >}}) or
+To learn more about sending native histograms to Mimir or Grafana Cloud Metrics via Grafana Alloy or Prometheus,
+see [Scrape and send native histograms with Grafana Alloy](https://grafana.com/docs/mimir/<MIMIR_VERSION>/send/native-histograms/#scrape-and-send-native-histograms-with-grafana-alloy) or
```

Comment:
I don't think we should be using absolute URLs here. Probably keep the existing `relref` links?

Reply:
relrefs are deprecated.  https://grafana.com/docs/writers-toolkit/write/shortcodes/#relref  They don't provide all the functionality we need within the docs. We prefer fully qualified URLs now (relative URLs also work).

Example:

* `https://grafana.com/docs/mimir/<MIMIR_VERSION>/send/native-histograms/#scrape-and-send-native-histograms-with-grafana-alloy` as the fully qualified URL
* `../../send/native-histograms/#scrape-and-send-native-histograms-with-grafana-alloy` as a relative URL